plugin/trino-delta-lake/pom.xml

```diff
@@ -366,4 +366,47 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
This profile will be disabled when any other profile is enabled, not only `cloud-tests`. Is this ok?

Reply:
I didn't know this, thanks for sharing. However, I think this is how we do test segregation in other modules, like hive-hadoop2, iceberg, bigquery, elasticsearch. I don't think Delta Lake is a any special.

And, i think it's safe. If `default` doesn't get activated, the worst thing that would happen is that we attempt to run all the tests, with no exclusions.